tests/testthat/test-warnings.R

```diff
@@ -0,0 +1,44 @@
+library(admiral.test)
```

Comment:
Can you update these tests so they follow our unit test guidance for numbering.  
![image](https://user-images.githubusercontent.com/10111024/180323731-9d3cf724-fec0-446b-8c56-e8fbef240cf0.png)

Reply:
You might be able to use the testthat adding that we created in admiraldev for just this purpose.